shared/triage_metadata.go

```diff
@@ -108,10 +108,11 @@ func (tm triageMetadata) getCommitBranchRef() (ref *github.Reference, err error)
 func (tm triageMetadata) getTree(ref *github.Reference, triagedMetadataMap map[string][]byte) (tree *github.Tree, err error) {
 	client := tm.githubClient
 
-	entries := []github.TreeEntry{}
+	entries := []*github.TreeEntry{}
 	for folderPath, content := range triagedMetadataMap {
 		dest := GetMetadataFilePath(folderPath)
-		entries = append(entries, github.TreeEntry{Path: github.String(dest), Type: github.String("blob"), Content: github.String(string(content)), Mode: github.String("100644")})
+		newEntry := github.TreeEntry{Path: github.String(dest), Type: github.String("blob"), Content: github.String(string(content)), Mode: github.String("100644")}
```

Comment:
Not too sure whether newEntry is needed here - I was getting "literal is not a type" errors when trying to inline creation of this TreeEntry point inside the call to append, and I couldn't really make sense of it. Open to suggestions.

Reply:
That's interesting. Is that the full error message?

For example, this works: https://play.golang.org/p/aNPerJmBrRS

Reply:
Ahh, ok, I was just thinking of this in C++ style (where taking the address of a stack-allocated variable is bad news). I was calling `append(entries, new(github.TreeEntry{...` which was throwing that error. Updated to your suggestion, which seems to work.

Reply:
Right, `new` only takes a type. Stack and heap are basically hidden away in Golang by GC and the memory model is closer to Java. Even if the object was indeed created on the stack, GC would be responsible for moving it out and updating all referencing pointers.

Just keep in mind don't cast the pointer to untyped `uintptr`, at which point there'll be no guarantees.